src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -404,10 +404,10 @@ void vtkPlusWinProbeVideoSource::FrameCallback(int length, char* data, char* hHe
         int slicePitch;
         int rowPitch;
         int tLength = WPDXGetFusedTexData(&texture, &slicePitch, &rowPitch);
-        assert(tLength == frameSize[0] * frameSize[1]);
+        //assert(tLength == frameSize[0] * frameSize[1]);
```

Comment:
If this line was uncommented, would it still make sense? Or is this obsolete code that can be removed?

Reply:
Yes, if it is important to check this then add proper error handling (log message, return with error, etc.), if not important then just remove the comment.

Reply:
Right, good catch.  The assert as is no longer makes sense, as the texture returned from the API is no longer guaranteed to have the same size as the image frame.  The updated check would be:  tLength == frameSize[0] * rowPitch.  Will put in proper error handling for this and push another commit.

Reply:
Tested recent assert update with hardware, LGTM